### PR TITLE
New code refactor

### PR DESCRIPTION
**Pull request by DIOHz0r**
### Changes description

New code refactor for the source code of the plugin

### References

Ref #503

### CODE REVIEW

File: inc/agent.class.php [hunk 1]
```diff
@@ -488,12 +487,12 @@ public function prepareInputForUpdate($input) {
 
       // send wipe to the agent
       if (isset($input['wipe']) && $input['wipe'] != '0') {
-         $input['wipe'] == '1';
```
Comment (btry) [inline inc/agent.class.php]: What ? I wrote that ?

**Comment by btry** [comment]
I don't like much the style of affecting a variable and re-affect in a if statement.It causes 2 or 3 instructions. 

The style I adopted causes 1 or 2 instructions execution. 
Maybe should we search or benchmark the performance of both

**Comment by DIOHz0r** [comment]
Hi,

The idea of that change is for better code reading according to bets practices. We can also simply some of those if using the following syntax 

```php 
$var = ($a && $b)? $a : 0;
```

**Comment by btry** [comment]
I would agree to use the ternary operator. But we need to keep the condition easy to read. I took the habit to split it into 3 lines if it is complex

(Condition)
? Alternative1
: Alternative 2;

File: inc/agent.class.php (hunk 1 above)
Comment (DIOHz0r) [reply inc/agent.class.php]: yes you did and it has been there for a lot of time

**Pull request merged**